<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>714</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>892</width>
    <height>714</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Boxigg交易系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDockWidget" name="dwMain">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="dockWidgetContents_5"/>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="dwIndicators">
      <property name="windowTitle">
       <string>指标管理</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QTableView" name="tableView_6"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>GroupBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="dwStrategyMonitor">
      <property name="windowTitle">
       <string>策略监控</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2"/>
     </widget>
    </item>
    <item>
     <widget class="QDockWidget" name="dwTrade">
      <property name="floating">
       <bool>true</bool>
      </property>
      <property name="features">
       <set>QDockWidget::AllDockWidgetFeatures</set>
      </property>
      <property name="windowTitle">
       <string>交易</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_7">
       <property name="minimumSize">
        <size>
         <width>159</width>
         <height>199</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionLoginQuoteServer"/>
    <addaction name="actionOpenAccount"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuStrategy">
    <property name="title">
     <string>策略管理</string>
    </property>
    <addaction name="actionAaa"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>数据管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTA">
    <property name="title">
     <string>技术分析</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionTest1"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuStrategy"/>
   <addaction name="menuData"/>
   <addaction name="menuTA"/>
   <addaction name="menuTest"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarStrategy">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenStrategy"/>
   <addaction name="actionTestStrategy"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBarTeck">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dwAccountMonitor">
   <property name="maximumSize">
    <size>
     <width>892</width>
     <height>257</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>账户管理</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>账户</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableView" name="tblvwAccount"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="pbQryAccount">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSaveAccount">
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>持仓</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableView" name="tblvwHolding"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="pbQryHolding">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSaveHolding">
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_7">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>成交</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTableView" name="tblvwTrade"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="pbQryTrade">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSaveTrade">
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>未成交</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QTableView" name="tblveUntrade"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QPushButton" name="pbQryUnTrade">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSaveUnTrade">
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>委托</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QTableView" name="tblvwOrder"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="pbQryOrder">
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSaveOrder">
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwManager">
   <property name="windowTitle">
    <string>管理面板</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoginQuoteServer">
   <property name="text">
    <string>登录行情服务器</string>
   </property>
  </action>
  <action name="actionOpenAccount">
   <property name="text">
    <string>OpenAccount</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAaa">
   <property name="text">
    <string>参数优化</string>
   </property>
   <property name="toolTip">
    <string>参数优化</string>
   </property>
  </action>
  <action name="actionTest1">
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
  <action name="actionOpenStrategy">
   <property name="text">
    <string>OpenStrategy</string>
   </property>
  </action>
  <action name="actionTestStrategy">
   <property name="text">
    <string>TestStrategy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
